src/ccall.cpp

```diff
@@ -1951,7 +1947,7 @@ jl_cgval_t function_sig_t::emit_a_ccall(
             // something alloca'd above is SSA
             if (static_rt)
                 return mark_julia_slot(result, rt, NULL, tbaa_stack);
-            result = ctx.builder.CreateLoad(result);
+            result = ctx.builder.CreateLoad(cast<PointerType>(result->getType())->getElementType(), result);
```

Comment:
`getElementType` is deprecated and must not be used in new code

Reply:
This would require some restructuring of `emit_a_ccall` – I'd rather leave this to somebody familiar with the code once `getElementType` is actually removed.